source/includes/api/_dataset.md

```diff
@@ -2,72 +2,13 @@
-
-### ArcGIS Feature service
-
-**`(connectorType: 'rest', provider: 'featureservice')`** [![](https://www.arcgis.com/features/img/logo-esri.png)](https://www.arcgis.com/features/index.html)<br>
-ArcGIS for server is a Complete, Cloud-Based Mapping Platform.
-
-### Google Earth Engine
-
-**`(connectorType: 'rest', provider: 'gee')`** [![](https://earthengine.google.com/static/images/GoogleEarthEngine_Grey_108.png)](https://earthengine.google.com/)<br>
-Google Earth Engine combines a multi-petabyte catalog of satellite imagery and geospatial datasets with planetary-scale analysis capabilities and makes it available for scientists, researchers, and developers to detect changes, map trends, and quantify differences on the Earth's surface.
-
-#### Web Map Services ([WMS](http://www.opengeospatial.org/standards/wms))
-
-**`(connectorType: 'wms', provider: 'wms')`**
-
-WMS connector provides access to data served through [OGC WMS](http://www.opengeospatial.org/standards/wms) protocol standard.
-
-#### Rasdaman (Raster Data Manager)
-
-**`(connectorType: 'rest', provider: 'rasdaman')`**<br>
-[Rasdaman](http://www.rasdaman.com/) is a database with capabilities for storage, manipulation and retrieval of multidimensional arrays.
-
-### Internal storage connectors
-
-For data stored in our system.
-
-#### NEX-GDDP
-
-**`(connectorType: 'rest', provider: 'nexgddp')`**<br>
-The NASA Earth Exchange Global Daily Downscaled Projections ([NEX-GDDP](https://nex.nasa.gov/nex/projects/1356/)) dataset is comprised of downscaled climate scenarios for the globe that are derived from the General Circulation Model (GCM) runs conducted under the Coupled Model Intercomparison Project Phase 5 (CMIP5) and across two of the four greenhouse gas emissions scenarios known as Representative Concentration Pathways (RCPs).
-
-#### Comma-Separated Values (CSV)
-
-**`(connectorType: 'document', provider: 'csv')`**<br>
-Arbitrary Comma-Separated Values data
-
-#### Tab-Separated Values (TSV)
-
-**`(connectorType: 'document', provider: 'tsv')`**<br>
-Arbitrary tab-Separated Values data
-
-#### JavaScript Object Notation (JSON)
-
-**`(connectorType: 'document', provider: 'json')`**<br>
-Arbitrary [json](http://www.json.org/) structured data
-
-#### XML
-
-**`(connectorType: 'document', provider: 'xml')`**<br>
-Arbitrary [XML](https://www.w3.org/TR/2006/REC-xml11-20060816/) data documents
+Once you've read that section, you can come back here to learn more details about [using existing datasets](#getting-all-datasets) or [creating your own](#creating-a-dataset).
 
 ## Getting all datasets
 
-This endpoint will allow to get all datasets available in the API:
+This endpoint will allow to get a list of the datasets available in the API. You can customize the result in multiple ways, using the options described below.
```

Comment:
Suggested: This endpoint will allow **you** to get **the** list of datasets (...)
Or alternatively: This endpoint allows you to get the list of datasets (...)

I find that "the" might be more appropriate than "a" as it's the list of all available datasets, right?